pkg/localstorev2/storer.go

```diff
@@ -58,9 +58,11 @@ type UploadStore interface {
 	// users requests to pin the data, a new pinning collection is created.
 	Upload(ctx context.Context, pin bool, tagID uint64) (PutterSession, error)
 	// NewSession can be used to obtain a tag ID to use for a new Upload session.
-	NewSession() (uint64, error)
+	NewSession() (SessionInfo, error)
 	// GetSessionInfo will show the information about the session.
 	GetSessionInfo(tagID uint64) (SessionInfo, error)
```

Comment:
`GetSessionInfo` -> `Session`.
`DeleteSessionInfo` -> `DeleteSession`.
Also, some comments on those methods would be nice.